parsers/NG.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+"""Parser for the electricity grid of Nigeria"""
+
+import arrow
+import logging
+import requests
+from bs4 import BeautifulSoup
+
+
+LIVE_PRODUCTION_API_URL = "https://www.niggrid.org/GenerationLoadProfileBinary?readingDate={0}&readingTime={1}"
+TYPE_MAPPING = {"hydro": "hydro", "gas": "gas", "gas/steam": "gas", "steam": "gas"}
+
+
+def extract_name_tech(company):
+    parts = company.split("(")
+    tech = parts[1].strip(")").lower()
+
+    return parts[0], TYPE_MAPPING[tech]
+
+
+def template_response(zone_key, datetime, source):
+    return {
+        "zoneKey": zone_key,
+        "datetime": datetime,
+        "production": {
+            "biomass": 0.0,
+            "coal": 0.0,
+            "gas": 0.0,
+            "hydro": 0.0,
+            "nuclear": 0.0,
+            "oil": 0.0,
+            "solar": 0.0,
```

Comment:
Given that the IEA reports [solar generation for Nigeria](https://www.iea.org/data-and-statistics?country=NIGERIA&fuel=Energy%20supply&indicator=ElecGenByFuel "chart is broken, but the Download chart data button works") but it's missing from the generation output, it might be better to omit this or set it to None. That way it'll show up as a question mark on the ElectricityMap site

![?](https://user-images.githubusercontent.com/2470175/102408558-0eeb4c80-3fee-11eb-8af7-7d68368756bc.png)

Reply:
* I've removed all of the extra production types which were not used
* Zones updated
* DATA_SOURCES updated